FAQ.md

```diff
@@ -7,5 +7,14 @@ You can either make a github issue in the bot repo: [gc/oldschoolbot](https://gi
 #### I can't tell which items are which in my bank, how do I see what they are?
 You can use `+bank --text` or `+bank --text --full` to see the names of your items.
 
+#### How do I search for specific items in my bank?
+You can use ``+bank --search=itemname`` to search for items.
+
 #### How do I create a Barrows set, or deconstruct a Barrows set into the items?
 Here's an example: `+create Dharoks armour set` to turn items into a set, or `+create Dharoks` to turn it back into the items.
+
+#### How do boosts work?
+Refer to [this](https://cdn.discordapp.com/attachments/685237018170622077/701220865173487616/image0.jpg) image for more information on boosts
```

Comment:
i think we should avoid linking to images and instead link to sections in the minions page where the information is written it

Reply:
I can rewrite Wyat's image in the boosts section.  Give me a few minutes.

Reply:
This has been changed, and links to the minion page now, under the boosts section.  Detailed explanation and examples are here https://github.com/gc/obdocs/pull/14